src/encoded/static/libs/formatMeasurement.js

```diff
@@ -0,0 +1,28 @@
+
+/**
+* Takes a magnitude and unit (singular form) and returns the measurement
+* with the proper unit in singular or plural, depending on magnitude.
+* Invalid measurements are turned as- Unknown
+* @param {number} magnitude - Measured value
+* @param {string} unit - Measurement system in singular form
+*/
+const formatMeasurement = (magnitude, unit) => {
+    // parseFloat used because parseInt only keeps integer
+    const magnitudeAsFloat = parseFloat(magnitude);
+
+    if (Number.isNaN(magnitudeAsFloat)) {
+        return 'Unknown';
+    }
+
+    if (magnitudeAsFloat === 0) {
+        return '0';
+    }
+
+    const measurement = `${magnitudeAsFloat} ${unit || ''}`.trim();
+
```

Comment:
Seems odd to me to add a space deliberately, then trim it if not needed. Couldn’t it be ``${unit ? `${unit}` : ''}``? I doubt there’s any real performance difference, but it just seems odd.